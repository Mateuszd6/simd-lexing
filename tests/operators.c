int main(int i1 = 0, int i2 = 0, int i3 = 0,
         int i4 = 0, int i5 = 0, int i6 = 0,
         int i7 = 0, int i8 = 0, int i9 = 0)
{
    long; long; long; long; long; long; long; long; long; long;
    long; long; long; long; long; long; long; long; long; long;
    long; long; long; long; long; long; long; long; long; long;

    ++ -- -> << >> && || <= >= == != += -= *= /= %= &= ^= |= ?:  ++ -- -> << >> && || <= >= == != += -= *= /= %= &= ^= |= ?:++ -- -> << >> && || <= >= == != += -= *= /= %= &= ^= |= ?:  ++ -- -> << >> && || <= >= == != += -= *= /= %= &= ^= |= ?:

    ++-->>
    ^^^

    >>== <<= ...      long; long; long; lo >>=
     >>== <<= ...      long; long; long; lo >>=       long; lo<<=


    long; long; long; long; long; long; long; long; long; long;
    long; long; long; long; long; long; long; long; long; long;
    long; long; long; long; long; long; long; long; long; long;

    return q;
}
