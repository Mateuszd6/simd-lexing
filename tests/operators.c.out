../tests/operators.c:1:1: TOK 0 (L = 3): "int"
../tests/operators.c:1:5: TOK 0 (L = 4): "main"
../tests/operators.c:1:9: TOK 1 (L = 1): "("
../tests/operators.c:1:10: TOK 0 (L = 3): "int"
../tests/operators.c:1:14: TOK 0 (L = 2): "i1"
../tests/operators.c:1:17: TOK 1 (L = 1): "="
../tests/operators.c:1:19: TOK 2 (L = 1): "0"
../tests/operators.c:1:20: TOK 1 (L = 1): ","
../tests/operators.c:1:22: TOK 0 (L = 3): "int"
../tests/operators.c:1:26: TOK 0 (L = 2): "i2"
../tests/operators.c:1:29: TOK 1 (L = 1): "="
../tests/operators.c:1:31: TOK 2 (L = 1): "0"
../tests/operators.c:1:32: TOK 1 (L = 1): ","
../tests/operators.c:1:34: TOK 0 (L = 3): "int"
../tests/operators.c:1:38: TOK 0 (L = 2): "i3"
../tests/operators.c:1:41: TOK 1 (L = 1): "="
../tests/operators.c:1:43: TOK 2 (L = 1): "0"
../tests/operators.c:1:44: TOK 1 (L = 1): ","
../tests/operators.c:2:10: TOK 0 (L = 3): "int"
../tests/operators.c:2:14: TOK 0 (L = 2): "i4"
../tests/operators.c:2:17: TOK 1 (L = 1): "="
../tests/operators.c:2:19: TOK 2 (L = 1): "0"
../tests/operators.c:2:20: TOK 1 (L = 1): ","
../tests/operators.c:2:22: TOK 0 (L = 3): "int"
../tests/operators.c:2:26: TOK 0 (L = 2): "i5"
../tests/operators.c:2:29: TOK 1 (L = 1): "="
../tests/operators.c:2:31: TOK 2 (L = 1): "0"
../tests/operators.c:2:32: TOK 1 (L = 1): ","
../tests/operators.c:2:34: TOK 0 (L = 3): "int"
../tests/operators.c:2:38: TOK 0 (L = 2): "i6"
../tests/operators.c:2:41: TOK 1 (L = 1): "="
../tests/operators.c:2:43: TOK 2 (L = 1): "0"
../tests/operators.c:2:44: TOK 1 (L = 1): ","
../tests/operators.c:3:10: TOK 0 (L = 3): "int"
../tests/operators.c:3:14: TOK 0 (L = 2): "i7"
../tests/operators.c:3:17: TOK 1 (L = 1): "="
../tests/operators.c:3:19: TOK 2 (L = 1): "0"
../tests/operators.c:3:20: TOK 1 (L = 1): ","
../tests/operators.c:3:22: TOK 0 (L = 3): "int"
../tests/operators.c:3:26: TOK 0 (L = 2): "i8"
../tests/operators.c:3:29: TOK 1 (L = 1): "="
../tests/operators.c:3:31: TOK 2 (L = 1): "0"
../tests/operators.c:3:32: TOK 1 (L = 1): ","
../tests/operators.c:3:34: TOK 0 (L = 3): "int"
../tests/operators.c:3:38: TOK 0 (L = 2): "i9"
../tests/operators.c:3:41: TOK 1 (L = 1): "="
../tests/operators.c:3:43: TOK 2 (L = 1): "0"
../tests/operators.c:3:44: TOK 1 (L = 1): ")"
../tests/operators.c:4:1: TOK 1 (L = 1): "{"
../tests/operators.c:5:5: TOK 0 (L = 4): "long"
../tests/operators.c:5:9: TOK 1 (L = 1): ";"
../tests/operators.c:5:11: TOK 0 (L = 4): "long"
../tests/operators.c:5:15: TOK 1 (L = 1): ";"
../tests/operators.c:5:17: TOK 0 (L = 4): "long"
../tests/operators.c:5:21: TOK 1 (L = 1): ";"
../tests/operators.c:5:23: TOK 0 (L = 4): "long"
../tests/operators.c:5:27: TOK 1 (L = 1): ";"
../tests/operators.c:5:29: TOK 0 (L = 4): "long"
../tests/operators.c:5:33: TOK 1 (L = 1): ";"
../tests/operators.c:5:35: TOK 0 (L = 4): "long"
../tests/operators.c:5:39: TOK 1 (L = 1): ";"
../tests/operators.c:5:41: TOK 0 (L = 4): "long"
../tests/operators.c:5:45: TOK 1 (L = 1): ";"
../tests/operators.c:5:47: TOK 0 (L = 4): "long"
../tests/operators.c:5:51: TOK 1 (L = 1): ";"
../tests/operators.c:5:53: TOK 0 (L = 4): "long"
../tests/operators.c:5:57: TOK 1 (L = 1): ";"
../tests/operators.c:5:59: TOK 0 (L = 4): "long"
../tests/operators.c:5:63: TOK 1 (L = 1): ";"
../tests/operators.c:6:5: TOK 0 (L = 4): "long"
../tests/operators.c:6:9: TOK 1 (L = 1): ";"
../tests/operators.c:6:11: TOK 0 (L = 4): "long"
../tests/operators.c:6:15: TOK 1 (L = 1): ";"
../tests/operators.c:6:17: TOK 0 (L = 4): "long"
../tests/operators.c:6:21: TOK 1 (L = 1): ";"
../tests/operators.c:6:23: TOK 0 (L = 4): "long"
../tests/operators.c:6:27: TOK 1 (L = 1): ";"
../tests/operators.c:6:29: TOK 0 (L = 4): "long"
../tests/operators.c:6:33: TOK 1 (L = 1): ";"
../tests/operators.c:6:35: TOK 0 (L = 4): "long"
../tests/operators.c:6:39: TOK 1 (L = 1): ";"
../tests/operators.c:6:41: TOK 0 (L = 4): "long"
../tests/operators.c:6:45: TOK 1 (L = 1): ";"
../tests/operators.c:6:47: TOK 0 (L = 4): "long"
../tests/operators.c:6:51: TOK 1 (L = 1): ";"
../tests/operators.c:6:53: TOK 0 (L = 4): "long"
../tests/operators.c:6:57: TOK 1 (L = 1): ";"
../tests/operators.c:6:59: TOK 0 (L = 4): "long"
../tests/operators.c:6:63: TOK 1 (L = 1): ";"
../tests/operators.c:7:5: TOK 0 (L = 4): "long"
../tests/operators.c:7:9: TOK 1 (L = 1): ";"
../tests/operators.c:7:11: TOK 0 (L = 4): "long"
../tests/operators.c:7:15: TOK 1 (L = 1): ";"
../tests/operators.c:7:17: TOK 0 (L = 4): "long"
../tests/operators.c:7:21: TOK 1 (L = 1): ";"
../tests/operators.c:7:23: TOK 0 (L = 4): "long"
../tests/operators.c:7:27: TOK 1 (L = 1): ";"
../tests/operators.c:7:29: TOK 0 (L = 4): "long"
../tests/operators.c:7:33: TOK 1 (L = 1): ";"
../tests/operators.c:7:35: TOK 0 (L = 4): "long"
../tests/operators.c:7:39: TOK 1 (L = 1): ";"
../tests/operators.c:7:41: TOK 0 (L = 4): "long"
../tests/operators.c:7:45: TOK 1 (L = 1): ";"
../tests/operators.c:7:47: TOK 0 (L = 4): "long"
../tests/operators.c:7:51: TOK 1 (L = 1): ";"
../tests/operators.c:7:53: TOK 0 (L = 4): "long"
../tests/operators.c:7:57: TOK 1 (L = 1): ";"
../tests/operators.c:7:59: TOK 0 (L = 4): "long"
../tests/operators.c:7:63: TOK 1 (L = 1): ";"
../tests/operators.c:9:5: TOK 1 (L = 2): "++"
../tests/operators.c:9:8: TOK 1 (L = 2): "--"
../tests/operators.c:9:11: TOK 1 (L = 2): "->"
../tests/operators.c:9:14: TOK 1 (L = 2): "<<"
../tests/operators.c:9:17: TOK 1 (L = 2): ">>"
../tests/operators.c:9:20: TOK 1 (L = 2): "&&"
../tests/operators.c:9:23: TOK 1 (L = 2): "||"
../tests/operators.c:9:26: TOK 1 (L = 2): "<="
../tests/operators.c:9:29: TOK 1 (L = 2): ">="
../tests/operators.c:9:32: TOK 1 (L = 2): "=="
../tests/operators.c:9:35: TOK 1 (L = 2): "!="
../tests/operators.c:9:38: TOK 1 (L = 2): "+="
../tests/operators.c:9:41: TOK 1 (L = 2): "-="
../tests/operators.c:9:44: TOK 1 (L = 2): "*="
../tests/operators.c:9:47: TOK 1 (L = 2): "/="
../tests/operators.c:9:50: TOK 1 (L = 2): "%="
../tests/operators.c:9:53: TOK 1 (L = 2): "&="
../tests/operators.c:9:56: TOK 1 (L = 2): "^="
../tests/operators.c:9:59: TOK 1 (L = 2): "|="
../tests/operators.c:9:62: TOK 1 (L = 2): "?:"
../tests/operators.c:9:66: TOK 1 (L = 2): "++"
../tests/operators.c:9:69: TOK 1 (L = 2): "--"
../tests/operators.c:9:72: TOK 1 (L = 2): "->"
../tests/operators.c:9:75: TOK 1 (L = 2): "<<"
../tests/operators.c:9:78: TOK 1 (L = 2): ">>"
../tests/operators.c:9:81: TOK 1 (L = 2): "&&"
../tests/operators.c:9:84: TOK 1 (L = 2): "||"
../tests/operators.c:9:87: TOK 1 (L = 2): "<="
../tests/operators.c:9:90: TOK 1 (L = 2): ">="
../tests/operators.c:9:93: TOK 1 (L = 2): "=="
../tests/operators.c:9:96: TOK 1 (L = 2): "!="
../tests/operators.c:9:99: TOK 1 (L = 2): "+="
../tests/operators.c:9:102: TOK 1 (L = 2): "-="
../tests/operators.c:9:105: TOK 1 (L = 2): "*="
../tests/operators.c:9:108: TOK 1 (L = 2): "/="
../tests/operators.c:9:111: TOK 1 (L = 2): "%="
../tests/operators.c:9:114: TOK 1 (L = 2): "&="
../tests/operators.c:9:117: TOK 1 (L = 2): "^="
../tests/operators.c:9:120: TOK 1 (L = 2): "|="
../tests/operators.c:9:123: TOK 1 (L = 2): "?:"
../tests/operators.c:9:125: TOK 1 (L = 2): "++"
../tests/operators.c:9:128: TOK 1 (L = 2): "--"
../tests/operators.c:9:131: TOK 1 (L = 2): "->"
../tests/operators.c:9:134: TOK 1 (L = 2): "<<"
../tests/operators.c:9:137: TOK 1 (L = 2): ">>"
../tests/operators.c:9:140: TOK 1 (L = 2): "&&"
../tests/operators.c:9:143: TOK 1 (L = 2): "||"
../tests/operators.c:9:146: TOK 1 (L = 2): "<="
../tests/operators.c:9:149: TOK 1 (L = 2): ">="
../tests/operators.c:9:152: TOK 1 (L = 2): "=="
../tests/operators.c:9:155: TOK 1 (L = 2): "!="
../tests/operators.c:9:158: TOK 1 (L = 2): "+="
../tests/operators.c:9:161: TOK 1 (L = 2): "-="
../tests/operators.c:9:164: TOK 1 (L = 2): "*="
../tests/operators.c:9:167: TOK 1 (L = 2): "/="
../tests/operators.c:9:170: TOK 1 (L = 2): "%="
../tests/operators.c:9:173: TOK 1 (L = 2): "&="
../tests/operators.c:9:176: TOK 1 (L = 2): "^="
../tests/operators.c:9:179: TOK 1 (L = 2): "|="
../tests/operators.c:9:182: TOK 1 (L = 2): "?:"
../tests/operators.c:9:186: TOK 1 (L = 2): "++"
../tests/operators.c:9:189: TOK 1 (L = 2): "--"
../tests/operators.c:9:192: TOK 1 (L = 2): "->"
../tests/operators.c:9:195: TOK 1 (L = 2): "<<"
../tests/operators.c:9:198: TOK 1 (L = 2): ">>"
../tests/operators.c:9:201: TOK 1 (L = 2): "&&"
../tests/operators.c:9:204: TOK 1 (L = 2): "||"
../tests/operators.c:9:207: TOK 1 (L = 2): "<="
../tests/operators.c:9:210: TOK 1 (L = 2): ">="
../tests/operators.c:9:213: TOK 1 (L = 2): "=="
../tests/operators.c:9:216: TOK 1 (L = 2): "!="
../tests/operators.c:9:219: TOK 1 (L = 2): "+="
../tests/operators.c:9:222: TOK 1 (L = 2): "-="
../tests/operators.c:9:225: TOK 1 (L = 2): "*="
../tests/operators.c:9:228: TOK 1 (L = 2): "/="
../tests/operators.c:9:231: TOK 1 (L = 2): "%="
../tests/operators.c:9:234: TOK 1 (L = 2): "&="
../tests/operators.c:9:237: TOK 1 (L = 2): "^="
../tests/operators.c:9:240: TOK 1 (L = 2): "|="
../tests/operators.c:9:243: TOK 1 (L = 2): "?:"
../tests/operators.c:11:5: TOK 1 (L = 2): "++"
../tests/operators.c:11:7: TOK 1 (L = 2): "--"
../tests/operators.c:11:9: TOK 1 (L = 2): ">>"
../tests/operators.c:12:5: TOK 1 (L = 1): "^"
../tests/operators.c:12:6: TOK 1 (L = 1): "^"
../tests/operators.c:12:7: TOK 1 (L = 1): "^"
../tests/operators.c:14:5: TOK 1 (L = 3): ">>="
../tests/operators.c:14:8: TOK 1 (L = 1): "="
../tests/operators.c:14:10: TOK 1 (L = 3): "<<="
../tests/operators.c:14:14: TOK 1 (L = 3): "..."
../tests/operators.c:14:23: TOK 0 (L = 4): "long"
../tests/operators.c:14:27: TOK 1 (L = 1): ";"
../tests/operators.c:14:29: TOK 0 (L = 4): "long"
../tests/operators.c:14:33: TOK 1 (L = 1): ";"
../tests/operators.c:14:35: TOK 0 (L = 4): "long"
../tests/operators.c:14:39: TOK 1 (L = 1): ";"
../tests/operators.c:14:41: TOK 0 (L = 2): "lo"
../tests/operators.c:14:44: TOK 1 (L = 3): ">>="
../tests/operators.c:15:6: TOK 1 (L = 3): ">>="
../tests/operators.c:15:9: TOK 1 (L = 1): "="
../tests/operators.c:15:11: TOK 1 (L = 3): "<<="
../tests/operators.c:15:15: TOK 1 (L = 3): "..."
../tests/operators.c:15:24: TOK 0 (L = 4): "long"
../tests/operators.c:15:28: TOK 1 (L = 1): ";"
../tests/operators.c:15:30: TOK 0 (L = 4): "long"
../tests/operators.c:15:34: TOK 1 (L = 1): ";"
../tests/operators.c:15:36: TOK 0 (L = 4): "long"
../tests/operators.c:15:40: TOK 1 (L = 1): ";"
../tests/operators.c:15:42: TOK 0 (L = 2): "lo"
../tests/operators.c:15:45: TOK 1 (L = 3): ">>="
../tests/operators.c:15:55: TOK 0 (L = 4): "long"
../tests/operators.c:15:59: TOK 1 (L = 1): ";"
../tests/operators.c:15:61: TOK 0 (L = 2): "lo"
../tests/operators.c:15:63: TOK 1 (L = 3): "<<="
../tests/operators.c:18:5: TOK 0 (L = 4): "long"
../tests/operators.c:18:9: TOK 1 (L = 1): ";"
../tests/operators.c:18:11: TOK 0 (L = 4): "long"
../tests/operators.c:18:15: TOK 1 (L = 1): ";"
../tests/operators.c:18:17: TOK 0 (L = 4): "long"
../tests/operators.c:18:21: TOK 1 (L = 1): ";"
../tests/operators.c:18:23: TOK 0 (L = 4): "long"
../tests/operators.c:18:27: TOK 1 (L = 1): ";"
../tests/operators.c:18:29: TOK 0 (L = 4): "long"
../tests/operators.c:18:33: TOK 1 (L = 1): ";"
../tests/operators.c:18:35: TOK 0 (L = 4): "long"
../tests/operators.c:18:39: TOK 1 (L = 1): ";"
../tests/operators.c:18:41: TOK 0 (L = 4): "long"
../tests/operators.c:18:45: TOK 1 (L = 1): ";"
../tests/operators.c:18:47: TOK 0 (L = 4): "long"
../tests/operators.c:18:51: TOK 1 (L = 1): ";"
../tests/operators.c:18:53: TOK 0 (L = 4): "long"
../tests/operators.c:18:57: TOK 1 (L = 1): ";"
../tests/operators.c:18:59: TOK 0 (L = 4): "long"
../tests/operators.c:18:63: TOK 1 (L = 1): ";"
../tests/operators.c:19:5: TOK 0 (L = 4): "long"
../tests/operators.c:19:9: TOK 1 (L = 1): ";"
../tests/operators.c:19:11: TOK 0 (L = 4): "long"
../tests/operators.c:19:15: TOK 1 (L = 1): ";"
../tests/operators.c:19:17: TOK 0 (L = 4): "long"
../tests/operators.c:19:21: TOK 1 (L = 1): ";"
../tests/operators.c:19:23: TOK 0 (L = 4): "long"
../tests/operators.c:19:27: TOK 1 (L = 1): ";"
../tests/operators.c:19:29: TOK 0 (L = 4): "long"
../tests/operators.c:19:33: TOK 1 (L = 1): ";"
../tests/operators.c:19:35: TOK 0 (L = 4): "long"
../tests/operators.c:19:39: TOK 1 (L = 1): ";"
../tests/operators.c:19:41: TOK 0 (L = 4): "long"
../tests/operators.c:19:45: TOK 1 (L = 1): ";"
../tests/operators.c:19:47: TOK 0 (L = 4): "long"
../tests/operators.c:19:51: TOK 1 (L = 1): ";"
../tests/operators.c:19:53: TOK 0 (L = 4): "long"
../tests/operators.c:19:57: TOK 1 (L = 1): ";"
../tests/operators.c:19:59: TOK 0 (L = 4): "long"
../tests/operators.c:19:63: TOK 1 (L = 1): ";"
../tests/operators.c:20:5: TOK 0 (L = 4): "long"
../tests/operators.c:20:9: TOK 1 (L = 1): ";"
../tests/operators.c:20:11: TOK 0 (L = 4): "long"
../tests/operators.c:20:15: TOK 1 (L = 1): ";"
../tests/operators.c:20:17: TOK 0 (L = 4): "long"
../tests/operators.c:20:21: TOK 1 (L = 1): ";"
../tests/operators.c:20:23: TOK 0 (L = 4): "long"
../tests/operators.c:20:27: TOK 1 (L = 1): ";"
../tests/operators.c:20:29: TOK 0 (L = 4): "long"
../tests/operators.c:20:33: TOK 1 (L = 1): ";"
../tests/operators.c:20:35: TOK 0 (L = 4): "long"
../tests/operators.c:20:39: TOK 1 (L = 1): ";"
../tests/operators.c:20:41: TOK 0 (L = 4): "long"
../tests/operators.c:20:45: TOK 1 (L = 1): ";"
../tests/operators.c:20:47: TOK 0 (L = 4): "long"
../tests/operators.c:20:51: TOK 1 (L = 1): ";"
../tests/operators.c:20:53: TOK 0 (L = 4): "long"
../tests/operators.c:20:57: TOK 1 (L = 1): ";"
../tests/operators.c:20:59: TOK 0 (L = 4): "long"
../tests/operators.c:20:63: TOK 1 (L = 1): ";"
../tests/operators.c:22:5: TOK 0 (L = 6): "return"
../tests/operators.c:22:12: TOK 0 (L = 1): "q"
../tests/operators.c:22:13: TOK 1 (L = 1): ";"
../tests/operators.c:23:1: TOK 1 (L = 1): "}"
Parsed: 24 lines, 92 ids, 0 strings, 0 chars, 9 ints, 0 hex,   0 floats,    0 //s, 0 /**/s,   0 #foo
